src/Compilers/CSharp/Portable/Binder/Semantics/Operators/BinaryOperatorOverloadResolution.cs

```diff
@@ -843,7 +865,7 @@ private BetterResult BetterOperator(BinaryOperatorSignature op1, BinaryOperatorS
             Debug.Assert(op1.Priority.HasValue == op2.Priority.HasValue);
 
             // We use Priority as a tie-breaker to help match native compiler bugs.
-            if (op1.Priority.HasValue && op2.Priority.HasValue && op1.Priority.GetValueOrDefault() != op2.Priority.GetValueOrDefault())
+            if (op1.Priority.HasValue && op1.Priority.GetValueOrDefault() != op2.Priority.GetValueOrDefault())
```

Comment:
>if (op1.Priority.HasValue && op1.Priority.GetValueOrDefault() != op2.Priority.GetValueOrDefault()) [](start = 12, length = 98)

It is not clear what motivated this change. It looks like the check became asymmetrical, i.e., if op1.Priority is not null and op2.Priority is null, we enter the if, but not if the values are swapped.